go_sequencer/synchronizer/synchronizer.go

```diff
@@ -718,11 +718,11 @@ func (s *Synchronizer) rollupSync(ethBlock *common.Block) (*common.RollupData, e
 				"evtForgeBatch.BatchNum = (%v)",
 				s.stateDB.CurrentBatch(), batchNum))
 		}
-		if s.stateDB.AccountTree.Root().BigInt().Cmp(forgeBatchArgs.NewStRoot) != 0 {
-			return nil, tracerr.Wrap(fmt.Errorf("stateDB.MTRoot (%v) != "+
-				"forgeBatchArgs.NewStRoot (%v)",
-				s.stateDB.AccountTree.Root().BigInt(), forgeBatchArgs.NewStRoot))
-		}
+		// if s.stateDB.AccountTree.Root().BigInt().Cmp(forgeBatchArgs.NewStRoot) != 0 {
```

Comment:
Regarding the commented part here @kaiden-araki Was this intentional. 

If not can you explain the reason. As to me it seems like it's checking the root of account tree to be equal to that received from the `RollupForgeBatchArgs` contract. 

Let me know if i missed someting in between. 

Thanks